src/documentation/pages/components/date-picker/examples/_calendar-active.vue

```diff
@@ -0,0 +1,91 @@
+<template lang="pug">
+  <ComponentExample titleSize="h4" title="Active" id="calendar-active" :tabs="exampleTabs">
+    chi-date(value="05/15/2019" slot="example")
+    <pre class="language-html" slot="code-webcomponent">
+      <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+    </pre>
+    <Wrapper slot="code-htmlblueprint">
+      <JSNeeded />
+        .p--text.chi-tab__description
+          | Use class <code>-active</code> for the selected date
+      <pre class="language-html">
+        <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+      </pre>
+    </Wrapper>
+  </ComponentExample>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator';
+
+@Component({
+  data: () => {
+    return {
+      exampleTabs: [
+        {
+          active: true,
+          id: 'webcomponent',
+          label: 'Web component',
+        },
+        {
+          id: 'htmlblueprint',
+          label: 'HTML blueprint',
+        },
+      ],
+      codeSnippets: {
+        webcomponent: `<chi-date value="05/15/2019"></chi-date>`,
+        htmlblueprint: `<div class="chi-datepicker -week-starts-on-sun -month-starts-on-wed">
+  <div class="chi-datepicker__month-row">
+    <div class="prev"><i class="chi-icon icon-chevron-left -sm" aria-hidden="true"></i></div>
+    <div class="chi-datepicker__month">May 2019</div>
+    <div class="next"><i class="chi-icon icon-chevron-right -sm" aria-hidden="true"></i></div>
+  </div>
+  <div class="chi-datepicker__day-names">
```

Comment:
Could we generate these `div` dynamically? You can use the picker group component as a reference. 

https://github.com/CenturyLink/Chi/blob/master/src/documentation/pages/components/picker-group/examples/_sizes-lumen-centurylink.vue

Reply:
Yes, updated it whereever it is required.